proposal/nested-map-functions.md

```diff
@@ -0,0 +1,204 @@
+# Nested Map Functions: Draft 1.0
+
+*[(Issue)](https://github.com/sass/sass/issues/1739)*
+
+This proposal updates the built-in `sass:map` module to better support merging,
+setting, and getting items from nested maps.
+
+## Table of Contents
+
+* [Background](#background)
+* [Summary](#summary)
+* [Syntax](#syntax)
+  * [`get()`](#get)
+  * [`has-key()`](#has-key)
+  * [`set()`](#set)
+  * [`merge()`](#merge)
+
+## Background
+
+> This section is non-normative.
+
+The current map inspection and manipulation functions don't provide any built-in
+support for managing nested maps. Projects often build thir own tooling, but
+the results are inconsistent, and often slow.
+
+## Summary
```

Comment:
It's probably a good idea to mention which of these new behaviors will be accessible from the legacy global function names (presumably all updated behaviors for existing functions), and which will be restricted to the new module system (presumably only `map.set()`).